coordinator/src/services/translation/prescription/demographics.ts

```diff
@@ -72,21 +72,15 @@ export function convertAddress(fhirAddress: fhir.Address, fhirPath: string): cor
     fhirAddress.district,
     fhirAddress.state
   ].flat().filter(Boolean)
-  const hl7V3Address = new core.Address(convertAddressUse(fhirAddress.use, fhirAddress.type, fhirPath))
+  const hl7V3Address = new core.Address(convertAddressUse(fhirAddress.use, fhirPath))
   hl7V3Address.streetAddressLine = allAddressLines.map(line => new core.Text(line))
   if (fhirAddress.postalCode !== undefined){
     hl7V3Address.postalCode = new core.Text(fhirAddress.postalCode)
   }
   return hl7V3Address
 }
 
-function convertAddressUse(fhirAddressUse: string, fhirAddressType: string, fhirPath: string) {
-  if (fhirAddressUse === undefined && fhirAddressType === undefined){
-    return undefined
-  }
-  if (fhirAddressType === "postal") {
```

Comment:
why isthis being removed?

Reply:
Because it doesn't match the way PDS has mapped address types. Their API only looks at the address use, not the address type. We're making our API consistent with that

Reply:
okay just making sure its intended :+1: